modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/renderer/ImageGalleryInfoItemRenderer.java

```diff
@@ -146,6 +154,9 @@ public void render(
 	@Reference
 	private Portal _portal;
 
+	@Reference
```

Comment:
Why do you reference an Util class?
You can access method statically:
`PortalUtil.getLiferayPortletResponse()`

Anyway this should not be needed because you can do the same with _portal reference